src/Tizen.NUI/src/public/Transition/Transition.cs

```diff
@@ -26,10 +26,13 @@ namespace Tizen.NUI
     /// Transition class will be used as a property of Navigator.Transition.
```

Comment:
Maybe something like: See [](xref:Tizen.NUI.Components.Navigator.Transition) property of [](xref:Tizen.NUI.Components.Navigator).

Reply:
I think below 2 line is unnecessary.